libs/platform/src/lib/approval-flow/interfaces/approval-data-source.ts

```diff
@@ -5,52 +5,43 @@ import { ApprovalUser } from './approval-user';
 import { ApprovalTeam } from './approval-team';
 
 export interface ApprovalDataSource {
```

Comment:
My issue initially with this is that it does not implement DataSource (open, close) interface just like the other data sources.  I would reimplement this class. There needs to be a way way to listen for data changes which AF component listen. 

Or maybe two since we have actual two source teams, users. I would not try to adjust existing implementaion